downstairs/src/main.rs

```diff
@@ -60,7 +60,12 @@ enum Args {
         #[clap(short, long, name = "UUID", action)]
         uuid: Uuid,
 
-        #[clap(long, default_value = "false", action)]
+        #[clap(
+            long,
+            default_value = "false",
+            default_missing_value = "true",
+            action(clap::ArgAction::Set)
```

Comment:
So, I had to change how this argument was handled slightly to keep the tests working. We're explicitly passing a value to the argument whereas by default clap flags don't accept values. I opted to not change the test but I can change either

https://github.com/oxidecomputer/crucible/blob/8ea317a9dfee65d7c018a1fda89fdec63a560ef3/tools/test_up.sh#L83-L84

Reply:
Ug, I'm  confused by what clap wants here.
The default value is false, but if the default is missing, then it is true?

Reply:
Yup, exactly:

| Arg | Effect |
| ----- | ------- |
| [no flag] | false |
| --encrypted=false | false |
| --encrypted=true | true |
| --encrypted | true |

Looking again, I guess the last column there is new behaviour I added. I can remove or keep that if you'd like

Reply:
I don't like how Clap has named these things, but I understand it now.  thanks!